src/Service/ComponentenCatalogusService.php

```diff
@@ -351,17 +368,22 @@ public function createComponentWithData(ObjectEntity $repository, array $compone
     /**
      * Get all components of the given source.
      *
-     * @param Source $source         The given source
-     * @param array  $componentArray The component array
+     * @param ObjectEntity $component      The component object
+     * @param array        $componentArray The component array
+     * @param Source       $source         The given source
      *
-     * @return ObjectEntity|null
+     * @return ObjectEntity|null The updated component as object.
      * @throws \Exception
      */
-    public function updateComponentWithData(ObjectEntity $component, array $componentArray, Source $githubSource): ?ObjectEntity
+    public function updateComponentWithData(ObjectEntity $component, array $componentArray, Source $source): ?ObjectEntity
     {
         $organizationSchema = $this->resourceService->getSchema($this->configuration['organizationSchema'], 'open-catalogi/open-catalogi-bundle');
-        $componentSchema    = $this->resourceService->getSchema($this->configuration['componentSchema'], 'open-catalogi/open-catalogi-bundle');
         $componentMapping   = $this->resourceService->getMapping($this->configuration['componentencatalogusMapping'], 'open-catalogi/open-catalogi-bundle');
+        if ($componentMapping instanceof Mapping === false
+            || $organizationSchema instanceof Entity === false
+        ) {
```

Comment:
Could make this a 1 line if statement

Reply:
You could, but this is quite readable

Reply:
But also a very obvious if statement after the code above.
And more lines could also make it less readable overall.
I guess this depends on the situation en opinion of the writer. I don't think we have any clear code style we use saying something about this?

Reply:
Also could argue that putting != null instead of instanceof Mapping === false when we are sure the result is correct or null is more readable and should be used instead. 
This is something we once specifically talked about and agreed on btw (with Ruben) one day in the past